src/test/api_compatibility.ts

```diff
@@ -35,7 +36,7 @@ export function apiCompatibilityChecks(createTracer = () => new Tracer(), option
             });
 
             describe('inject', () => {
-                it('should not throw exception on required carrier types', () => {
+                options.checkInjectBehaviour ? it : it.skip('should not throw exception on required carrier types', () => {
```

Comment:
did you mean 

`(options.checkInjectBehaviour ? it : it.skip)('should not throw exception on required carrier type...`?

in the current format it looks like it just returns the `it` function without invoking it.

Reply:
Yes.